compiler/modules/print.llvm/llvm.bal

```diff
@@ -314,20 +323,74 @@ public class FunctionDefn {
         else {
             self.variableNames[regName] = 1;
         }
+        if name is string && !self.nameValid(regName) {
+            regName = "\"" + regName + "\"";
+        }
         string reg = "%" + regName;
         return reg;
-    } 
+    }
+
+    // Verify name is a valid LLVM name otherwise must escape
+    function nameValid(string name) returns boolean{
+        string nameLower = name.toLowerAscii();
+        // Currently check only the first character
+        string firstChar = nameLower.substring(0,1);
+        if firstChar == "_" || firstChar == "$" {
+            return true;
+        }
+        int first = firstChar.getCodePoint(0);
+        if firstChar >= "a" && firstChar <= "z" {
+            // First character is an alphabetic character
+            return true;
+        }
+        return false;
+    }
 
-    public function addEnumAttribute(EnumAttribute attribute){
+    public function addEnumAttribute(EnumAttribute attribute) {
         if self.attributes.indexOf(attribute) == () {
-           self.attributes.push(attribute); 
+            self.attributes.push(attribute);
```

Comment:
Extra space

Reply:
In the older one there was a space missing (only 3 instead of 4) there